lib/scrabble.rb

```diff
@@ -1,7 +1,17 @@
 class Scrabble
 
   def score(word)
-    1
+    point =""
```

Comment:
gotcha, sorry this took so long to reply. I'm working on weeding out all the spam in my email so it will unsub me from all the junk. I didn't see this until just now.